pkg/cmd/server/start/start_master.go

```diff
@@ -242,10 +242,10 @@ func (o MasterOptions) RunMaster() error {
 			return err
 		}
 
-		if err := os.MkdirAll(path.Dir(o.MasterArgs.GetConfigFileToWrite()), os.FileMode(0755)); err != nil {
```

Comment:
Is that common practice in the packages set up by the OS?  Do the etc
tls private key dirs do that?  I'm looking for a policy statement of
"this is normal"